### DISCUSSION TITLE
Pending rename.. [Jun 11]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2426] [Discord Threads]
> 🙋‍♂️ **λRick**

**Comment by o1lab** [comment]
> 🙋‍♂️ **FrancyMak**  
#### Clould you send me your request json?  
#  
> 🙋‍♂️ **λRick**  
#### `{"GitHub Stars": 2279, "updated_at": "2022-05-29 11:01"}`  
#  
> 🙋‍♂️ **FrancyMak**  
#### I am a recent arrival to the noco team, so I am still figuring out how their APIs work, and I would not want to give false guidance. Therefore, I will leave you to one of our software engineers who will be able to help you more than me.  
#  
> 🙋‍♂️ **FrancyMak**  
#### <@&682859083799199814>  
#  
> 🙋‍♂️ **FrancyMak**  
#### but, would be helpful, if you can send the path parametes that you're using (you can rename sensible content)  
#  
> 🙋‍♂️ **wkw**  
#### <@198536651348115466> which endpoint are you using?  
#  
> 🙋‍♂️ **λRick**  
#### /api/v1/db/data/noco/tools/tools/{rowId}  
#  
> 🙋‍♂️ **wkw**  
#### ok. thats dbTableRow.update. can u share your table schema  
#  
> 🙋‍♂️ **wkw**  
#### and how you send the request? if u r using curl, then share the full command with sensitive info redacted  
#  
> 🙋‍♂️ **λRick**  
#### ```{   
####   "A": "string",   
####   "B": "string",   
####   "C": "string",   
####   "D": "string",   
####   "E": "string",   
####   "F": "string",   
####   "G": "string",   
####   "H": "string",   
####   "I": "string",   
####   "J": "string",   
####   "K": "string",   
####   "L": "string",   
####   "M": "string",   
####   "N": "string",   
####   "O": "string",   
####   "P": "string",   
####   "Q": 0,   
####   "GitHub Stars": 0,   
####   "id": 0,   
####   "R": "string"   
#### }```  
#  
> 🙋‍♂️ **λRick**  
#### ```   
####    
#### resp = requests.patch(update_endpoint, data=json.dumps(delta_obj), headers={"xc-auth": xc_key, "Content-Type": "application/json"})   
#### ```  
#  
> 🙋‍♂️ **λRick**  
#### Any idea?  
#  
> 🙋‍♂️ **wkw**  
#### sorry. forget to press enter ...   
####    
#### may i confirm the following    
####    
#### - update_endpoint includes the row id   
#### - xc_key is auth token, not api token  
#  
> 🙋‍♂️ **λRick**  
#### Both yes  
#  
> 🙋‍♂️ **λRick**  
#### Found the cause, was using the HTTP instead of the HTTPS endpoint. Some of the endpoint API calls were working just fine on HTTP, and the error didn't point at this being the issue at all. What made me discover it was the Swagger UI in which the PATCH call did work.  
#  
> 🙋‍♂️ **λRick**  
#### Kept reducing differences until literally the last things left to try was the protocol  
#  
> 👨‍🦲 **dstala**  
#### <@198536651348115466> Thanks for updating the conclusion summary  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/980585095922589716)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
